vllm/config.py

```diff
@@ -739,6 +743,13 @@ def maybe_create_spec_config(
                 "num_speculative_tokens to be provided, but found "
                 f"{speculative_model=} and {num_speculative_tokens=}.")
 
+        if (speculative_disaable_by_enqueue_requests is not None
+                and speculative_disaable_by_enqueue_requests < 1):
```

Comment:
when would it make sense to have `speculative_disable_by_enqueue_requests == 1`? we should make it invalid to make the API easier to use (no footguns !)

Reply:
`1` doesn't make sense actually, but I'm not sure how small we should reject. Do you think `2` would make more sense?